### DISCUSSION TITLE
In the Code Server service, the webview in my extension plugin cannot be displayed. How to solve it?

### DISCUSSION THREAD

**Discussion by kevinKaikai** [coder/code-server#6656] [General]
![image](https://github.com/coder/code-server/assets/37619768/ca8c1f2a-97bc-479f-8bd3-aabb7b77df17)

Browser console error

![image](https://github.com/coder/code-server/assets/37619768/ec23c5ce-c35b-4af1-9055-e53e25b98d87)

**Comment by code-asher** [comment]
You have two options:

1. Access code-server over a secure context.  This means either `https` with a valid certificate or `localhost`.
2. Disable the secure context restriction in your browser. For example, Chrome has `unsafely-treat-insecure-origin-as-secure` but I am not sure about Firefox.
